oleggerasinovich/Titanic.ipynb

```diff
@@ -0,0 +1,1686 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    classified['Percentage'] = (classified['Survived'] / classified['Age']) * 100
</pre><p> </p><p>Instead of <code>Percentage</code>, you can name the column <code>survived_percentage</code> to make it more verbose.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/u2-de-students-lab/titanic_exploration/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>